.github/actions/csharp-bindings-action/action.yml

```diff
@@ -10,7 +10,7 @@ runs:
 
     - name: Setup ClangSharpPInvokeGenerator
       shell: bash
-      run: dotnet tool install --global ClangSharpPInvokeGenerator
+      run: dotnet tool install --global ClangSharpPInvokeGenerator --version 20.1.2.1
```

Comment:
Apparently, the latest version of `ClangSharpPInvokeGenerator` is not working, so we'll stick to the previous version: https://github.com/dotnet/ClangSharp/issues/625